ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -398,6 +319,21 @@ private void dropTablesExcludedInReplScope(ReplScope replScope) throws HiveExcep
             dbName);
   }
 
+  private void createReplLoadCompleteAckTask() {
+    if ((work.isIncrementalLoad() && !work.incrementalLoadTasksBuilder().hasMoreWork() && !work.hasBootstrapLoadTasks())
```

Comment:
During incremental, when we have no events in dump and only external tables are present, will the event folder still be created? If not the check work.incrementalLoadTasksBuilder().hasMoreWork() might not suffice?

Reply:
The check is already present before the method is being called